### DISCUSSION TITLE
Proxmox > Home Assistant OS VM > Slot A (OK=1 TRY=1)

### DISCUSSION THREAD

**Discussion by Lockie85** [tteck/Proxmox#1572] [Help Needed]
Hi,

My setup utilises the script:
`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/vm/haos-vm.sh)"`

Everything appeared to be working fine for about a week, then one morning within Home Assistant I did a quick check config, all green so I restarted and Home Assistant didn’t appear to come back. After checking Proxmox 8 all I see in the console is:
`Proxmox Slot A (OK=1 TRY=1)`

While it doesn't look like the system has correctly booted, I am able to access Home Assistant after a few full system reboots. But I still have the above issue in the Proxmox console. So something feels wrong, as previously I would see the Home Assistant terminal.

My System hardware is:
Device: Intel NUC 11
CPU: Intel Core i7-1165G7
RAM: 32 GB
Storage: 1TB Kingston PCIe SSD

My VM hardware setup is as follows:
![image](https://github.com/tteck/Proxmox/assets/7091582/5d6dd113-be3e-4998-9ec4-9d8d1020e19b)

Any help would be greatly appreciated. I’m very much a beginner with Proxmox, I know enough to get myself running but that’s about it. So please add additional details if you can around things I could look at doing.

Many Thanks.

**Comment by tteck** [comment]
To begin, I suggest removing the USB and PCI devices and checking if it successfully starts up.

**Reply by Lockie85** [reply]
Firstly many thanks for your response @tteck 

I shutdown the VM, removing both devices. Booted the VM and it fixed the issue. 

I then shutdown the VM, added back the USB Coral device. Booted the VM and the issue was still fixed.

I then shutdown the VM, added back the PCI device which is "_TigerLake-LP GT2 [Iris Xe Graphics]_". . Booted the VM and the issue was back!

So I assume for some reason the addition of the GPU is causing a problem? I've made a assumption that I should add the GPU so I can utilise it within Home Assistant and Frigate. But perhaps that's a bad/wrong idea?

Here's how I added it:
![image](https://github.com/tteck/Proxmox/assets/7091582/d0dcd73f-c72d-4d16-87d2-d51d9e2156cc)

![image](https://github.com/tteck/Proxmox/assets/7091582/05614dff-18cc-4460-954d-3b3594baf3d1)

![image](https://github.com/tteck/Proxmox/assets/7091582/78cd9e40-bec2-4c81-ad1c-09a73ef0d073)

Any thoughts?

**Comment by Lockie85** [comment]
In fact, without the addition of the GPU it seems my cameras have stopped working in Frigate. I think this is because I define hardware acceleration as "_hwaccel_args: preset-intel-qsv-h264_". So I might need to figure out how to added the GPU in without it causing issues, if you have any ideas on that?

I can remove that flag to get my cameras working again. But I _think_ that puts more stress on my CPU.

**Reply by tteck** [reply]
You could try experimenting with q35 machine type. 
> You can choose between the default Intel 440FX or the Q35 chipset, which also provides a virtual PCIe bus, and thus may be desired if one wants to pass through PCIe hardware.

https://pve.proxmox.com/wiki/Qemu/KVM_Virtual_Machines

Additionally, I recommend opting for ext4 rather than ZFS as the underlying file system for a small homelab setup.

**Reply by Lockie85** [reply]
I'll have a look at that.

Do you think there is any issue with the error I was getting. As I'm thinking, given I could still access Home Assistant. Perhaps I could just leave it. I don't need to access the console. I was only concerned it was going to cause additional issues. Might it, or should it be fine?

**Reply by tteck** [reply]
Personally, I take issue with any error.
